scripts/format_doc

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Should this be invoked by the `format` script now?

Reply:
It doesn't quite line up (`prettier` is also invoked for `.toml` and `.yaml` files) so I left them separate.